src/main/kotlin/build/buf/intellij/annotator/BufAnalyzeUtils.kt

```diff
@@ -204,7 +207,7 @@ object BufAnalyzeUtils {
         if (handler.waitFor(BUF_COMMAND_EXECUTION_TIMEOUT.toMillis())) {
             val code = exitCode.get()
             if (code != 0) {
-                LOG.warn("buf exit code: $code")
+                LOG.warn("${cmd.get() ?: "buf"} exit code: $code")
```

Comment:
Missed this as part of last PR - we should attempt to log the command and arguments.